<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSelectHardware</class>
 <widget class="QDialog" name="DialogSelectHardware">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>670</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>670</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Hardware Nvidia</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../imageResource.qrc">
    <normaloff>:/icons/cuda.png</normaloff>:/icons/cuda.png</iconset>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="1">
      <widget class="QComboBox" name="deviceComboBox">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Dispositivos Disponibles</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="deviceLabel">
       <property name="text">
        <string>Seleccione un dispositivo &amp;Nvidia </string>
       </property>
       <property name="buddy">
        <cstring>deviceComboBox</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="maximumSize">
      <size>
       <width>90</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Seleccionar dispositivo</string>
     </property>
     <property name="statusTip">
      <string>Dispositivos Disponibles</string>
     </property>
     <property name="text">
      <string>Seleccionar</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTableWidget" name="tableWidget">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="rowCount">
      <number>15</number>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column/>
     <column/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../imageResource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>DialogSelectHardware</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>660</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>512</x>
     <y>13</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
